File: tests/ai-fixtures/vibecheck-ai.php
<?php
// ruleid: vibecheck-untrusted-input-in-system-prompt-php
$m = ['role' => 'system', 'content' => $_GET['persona']];
function ex($client) {
  $resp = $client->chat()->create([]);
  // ruleid: vibecheck-llm-output-executed-php
  system($resp->content);
}
function htmlout($client) {
  $r = $client->chat()->create([]);
  // ruleid: vibecheck-llm-output-raw-html-php
  echo $r->content;
}

File: tests/ai-fixtures/vibecheck-ai.cs
public class F {
  void H() {
    // ruleid: vibecheck-untrusted-input-in-system-prompt-cs
    var m = new SystemChatMessage(Request.Query["persona"]);
  }
  void Ex(dynamic client, object msgs) {
    var resp = client.CompleteChat(msgs);
    // ruleid: vibecheck-llm-output-executed-cs
    Process.Start(resp.Content);
  }
  void Rh(dynamic client, object msgs) {
    var r = client.CompleteChat(msgs);
    // ruleid: vibecheck-llm-output-raw-html-cs
    Html.Raw(r.Content);
  }
}
